scripts/ci/run-ci-tests.sh

```diff
@@ -51,12 +52,17 @@ ci_prep () {
 	# This can fail on aarch64 travis
 	service apport stop || :
 
-	CC=gcc
-	# clang support
 	if [ "$CLANG" = "1" ]; then
-		CI_PKGS="$CI_PKGS clang"
+		# clang support
 		CC=clang
+		# If this is running in an environment without gcc installed
+		# compel-host-bin will fail as it is using HOSTCC. Also
+		# set HOSTCC to clang to build compel-host-bin with it.
+		export HOSTCC=clang
```

Comment:
I think this should be in a separate commit. It isn't about Drone...

Reply:
Created a separate commit.